src/Corrade/Utility/AndroidLogStreamBuffer.cpp

```diff
@@ -33,7 +34,10 @@ AndroidLogStreamBuffer::~AndroidLogStreamBuffer() = default;
 
 int AndroidLogStreamBuffer::sync() {
     /* Write the data */
-    __android_log_write(std::int32_t(_priority), _tag.data(), str().data());
+    const char * tag = _tag.data();
+    const char * data = str().data();
+    __android_log_write(std::int32_t(_priority), tag, data);
+    std::cout << data;
 
```

Comment:
This causes *all* output to be put into `std::cout`, with no chance of distincting between stdout or stderr. I'm not sure I like that. Make this configurable? Or choose it based on `_priority`?

Also, what about flushing? Usually when you have error or assertion messages, you'd want them to be printed right away and not in some buffer that might not get emptied before the application crashes / aborts.